Makefile

```diff
@@ -238,7 +269,7 @@ e2e-debug:
 # e2e test coverage
 ############################################################
 build-instrumented:
-	go test -covermode=atomic -coverpkg=github.com/open-cluster-management/$(IMG)... -c -tags e2e ./cmd/manager -o build/_output/bin/$(IMG)-instrumented
+	go test -covermode=atomic -coverpkg=github.com/open-cluster-management/$(IMG)... -c -tags e2e ./ -o build/_output/bin/$(IMG)-instrumented
```

Comment:
```suggestion
	go test -covermode=atomic -coverpkg=github.com/open-cluster-management-io/$(IMG)... -c -tags e2e ./ -o build/_output/bin/$(IMG)-instrumented
```

Reply:
I think the package/module name still does not have `-io`. And I think that's correct for now. Eventually I would want to move the module name to be separate from the exact github path, like https://github.com/open-cluster-management-io/registration/blob/main/go.mod#L1. Then when we make our "midstream" fork, we can do `replace` directives in our `go.mod` files to make things point to the correct repos as necessary, without too much mess around URLs.

Reply:
Ah, I see we haven't updated the `go.mod` file in this repo yet. Why hasn't the module name been changed? How would reference this repo vs midstream if the module name is the same?

Reply:
Via `replace`: https://golang.org/ref/mod#go-mod-file-replace